web/src/features/map/map-layers/GridAlertsLayer.tsx

```diff
@@ -113,7 +113,10 @@ export default function GridAlertsLayer() {
 
         return (
           <Marker key={zoneId} longitude={longitude} latitude={latitude} anchor="center">
-            <div className="flex items-center justify-center rounded-full bg-white/10 p-2">
+            <div
+              className="flex items-center justify-center rounded-full bg-white/10 p-2"
+              style={{ pointerEvents: 'none' }}
```

Comment:
I think you might need the pointer events none on the marker instead.
I should accept custom classes so just add the `pointer-events-none` tailwind class.